SETUP/INSTALL.md

```diff
@@ -319,12 +319,6 @@ hierarchy:
     d/teams
     d/teams/avatar
     d/teams/icon
-    d/pg
```

Comment:
Looking on both PROD and TEST, the `d/pg` directory still has up-to-date information in it. On PROD:
```
 6824667 Jun  1 06:03 catalog.txt
12809218 Jun  1 08:45 davids_list.html
12666517 Jun  1 08:45 davids_list_new.txt
```
So, there's something, somewhere, that's still using it.

Reply:
That's from `noncvs`. It is no longer required as part of the main code so it doesn't need to be in `INSTALL.md`.